examples/image_filtering/src/host/parameters/generic_line_parser.ads

```diff
@@ -0,0 +1,149 @@
+----------------------------------------------------------------------------
+--            Generic Command Line Parser (gclp)
+--
+--               Copyright (C) 2012, Riccardo Bernardini
```

Comment:
Ah I did not see that, my comments still stand though, this code seems fishy

Reply:
3rd party